src/lib.rs

```diff
@@ -40,7 +40,7 @@
 //! img.save("empty.jpg")?;
 //!
 //! let mut bytes: Vec<u8> = Vec::new();
-//! img2.write_to(&mut bytes, image::ImageOutputFormat::Png)?;
+//! img2.write_to(&mut Cursor::new(&mut bytes), image::ImageOutputFormat::Png)?;
```

Comment:
Do you think the use of the Cursor should have an explanation, or do we assume this is common knowledge?

Reply:
I'd say it is common enough. We might give readers a small nudge by making the `use` of `Cursor` visible.

Reply:
Sounds good. Any more changes before merging?